sidecar.sh

```diff
@@ -42,36 +41,63 @@ populate_notebook() {
 
 function populate() {
     log "querying manifest service at $GEN3_ENDPOINT/manifests/"
-    MANIFESTS=$(curl -s -H "Authorization: Bearer ${ACCESS_TOKEN}" "https://$GEN3_ENDPOINT/manifests/")
-    while [ -z "$MANIFESTS" ]; do
-        log "Unable to get manifests from '$GEN3_ENDPOINT/manifests/'"
-        log $MANIFESTS
+    MANIFEST=$(curl -s -H "Authorization: Bearer ${ACCESS_TOKEN}" "https://$GEN3_ENDPOINT/manifests/")
+    log "querying manifest service at $GEN3_ENDPOINT/metadata/"
+    METADATA=$(curl -s -H "Authorization: Bearer ${ACCESS_TOKEN}" "https://$GEN3_ENDPOINT/metadata/")
+
+    while [ -z "$MANIFEST" ] && [ -z "$METADATA_FILE" ]; do
+        if [ -z "$MANIFESTE" ]; then
```

Comment:
```suggestion
        if [ -z "$MANIFEST" ]; then
```
I think this is a typo